app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:email, :password, :password_confirmation, :current_password, :provider, :uid)
```

Comment:
I think you should not enable updating :provider and :uid

This fields must not be updatable from user interface